.gitignore

```diff
@@ -10,9 +10,9 @@ Cargo.lock
 **/*.rs.bk
 
 
-input/
```

Comment:
Why were these modified?

Reply:
To only match the top-level `input` directory, not the ones deeper down in the hierarchy.